doc/sphinx-guides/source/container/running/demo.rst

```diff
@@ -165,6 +165,53 @@ PID Providers
 
 Dataverse supports multiple Persistent ID (PID) providers. The ``compose.yml`` file uses the Permalink PID provider. Follow :ref:`pids-configuration` to reconfigure as needed.
 
+.. _additional-metadata-blocks:
+
+Additional Metadata Blocks
+++++++++++++++++++++++++++
+
+Metadata fields such as "Title" are part of a metadata block such as "Citation". See :ref:`metadata-references` in the User Guide for the metadata blocks that ship with Dataverse.
+
+At a high level, we will be loading a metadata block and then adjusting our Solr config to know about it.
+
+Care should be taken when adding additional metadata blocks. There is no way to `preview <https://github.com/IQSS/dataverse/issues/2551>`_ or `delete <https://github.com/IQSS/dataverse/issues/9628>`_ a metadata block so please use a throwaway environment.
+
+:ref:`metadata-references` lists some experimental metadata blocks. In the example below, we'll use the CodeMeta block.
+
+First, download a metadata block or create one by following :doc:`/admin/metadatacustomization` in the Admin Guide.
+
+Load the metadata block like this:
+
+``curl http://localhost:8080/api/admin/datasetfield/load -H "Content-type: text/tab-separated-values" -X POST --upload-file codemeta.tsv``
+
+Next, reconfigure Solr to know about the new metadata block.
+
+You can back up your existing Solr schema like this:
+
+``cp docker-dev-volumes/solr/data/data/collection1/conf/schema.xml docker-dev-volumes/solr/data/data/collection1/conf/schema.xml.orig``
+
+You can see the existing fields Solr know about like this:
+
+``curl http://localhost:8983/solr/collection1/schema/fields``
+
+Update your Solr schema with the following command:
+
+``curl http://localhost:8080/api/admin/index/solr/schema | docker run -i --rm -v ./docker-dev-volumes/solr/data:/var/solr gdcc/configbaker:unstable update-fields.sh /var/solr/data/collection1/conf/schema.xml``
```

Comment:
Consider removing the unstable Docker tag to emphasize that this approach is compatible with other image versions as well.